packages/xrpl/test/setupClient.ts

```diff
@@ -1,64 +1,92 @@
-/* eslint-disable no-param-reassign -- Necessary for test setup */
-/* eslint-disable @typescript-eslint/explicit-module-boundary-types -- Necessary for test setup */
 import { Client, BroadcastClient } from 'xrpl-local'
 
-import createMockRippled from './createMockRippled'
-import { getFreePort } from './testUtils'
+import createMockRippled, {
+  type MockedWebSocketServer,
+} from './createMockRippled'
+import { destroyServer, getFreePort } from './testUtils'
+
+export interface XrplTestContext {
+  client: Client | BroadcastClient
+  _mockedServerPort?: number
+  mockRippled?: MockedWebSocketServer
+  mocks?: MockedWebSocketServer[]
+  servers?: number[]
+}
 
 async function setupMockRippledConnection(
```

Comment:
Why the change from using `this.client` everywhere?

Reply:
`this` context only exists in mocha